domain/src/test/java/org/oppia/domain/oppialogger/analytics/AnalyticsControllerTest.kt

```diff
@@ -43,11 +64,26 @@ class AnalyticsControllerTest {
   @Inject
   lateinit var analyticsController: AnalyticsController
 
+  @Inject
+  lateinit var networkConnectionUtil: NetworkConnectionUtil
+
   @Inject
   lateinit var fakeEventLogger: FakeEventLogger
 
+  @InternalCoroutinesApi
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+
+  @Mock
+  lateinit var mockOppiaEventLogsObserver: Observer<AsyncResult<OppiaEventLogs>>
+
+  @Captor
+  lateinit var oppiaEventLogsResultCaptor: ArgumentCaptor<AsyncResult<OppiaEventLogs>>
+
   @Before
   fun setUp() {
+    MockitoAnnotations.initMocks(this)
```

Comment:
If you add the mockito junit rule, you shouldn't require this. I have had issues with it before though so in case that doesn't work then I guess this is okay. But let's first try using the mockito rule to avoid having this line at set up in all our tests!

Reply:
Yes, it worked. Thanks !